packages/app/src/app/components/SignInModal.tsx

```diff
@@ -47,10 +49,35 @@ export const SignInModal = () => {
               bottom: 0,
               zIndex: 9999,
               background: 'rgba(0, 0, 0, 0.75)',
+
+              // OutsideClickHandler styles
+              // can't use styled(OutsideClickHandler)
+              '> div': {
+                flexGrow: 1,
+                maxWidth: '720px',
+              },
             }}
           >
             <OutsideClickHandler onOutsideClick={toggleSignInModal}>
-              <SignInModalElement redirectTo={redirectOnLogin} />
+              <Element
+                css={css({
+                  boxSizing: 'border-box',
+                  backgroundColor: '#1D1D1D',
+                  borderRadius: '4px',
+                  overflow: 'hidden',
+                  margin: 'auto',
+                  padding: '24px',
+                  width: '100%',
+
+                  // With tokens
+                  boxShadow: '2',
```

Comment:
Do we use box shadow for modals in prism? I was thinking about what to do with things like animations, box shadows and other tokens that are a bit easier to sync between prism and the design system here, maybe it makes sense to keep the tokens with the same names (if possible) so the transition is smoother?

Reply:
I've checked the designs, tokens and components and what their shadow values are right now. I'll discuss with Lena what we should do here.

design:
```
0px 4px 4px rgba(0, 0, 0, 0.25);
```
v1 token 2:
```
0px 4px 4px rgba(0, 0, 0, 0.12), 0px 16px 32px rgba(0, 0, 0, 0.24)
```

in v2 prism we have a popup shadow, but no dialog shadow it seems:
```
0px 2px 7px rgba(0, 0, 0, 0.15), 0px -2px 7px rgba(0, 0, 0, 0.15), 0px 5px 17px rgba(0, 0, 0, 0.3)
```